library/unwind/src/lib.rs

```diff
@@ -42,6 +42,10 @@ cfg_if::cfg_if! {
 #[link(name = "gcc_s", cfg(not(target_feature = "crt-static")))]
 extern "C" {}
 
+#[cfg(all(target_os = "linux", target_env = "gnu", not(feature = "llvm-libunwind")))]
+#[link(name = "gcc_s", cfg(not(target_feature = "crt-static")))]
```

Comment:
Shouldn't it link `gcc_eh` with `crt-static`?

Reply:
Looks like `gcc_eh` is linked from libc (https://github.com/rust-lang/libc/pull/1899), perhaps due to linking order?
(I agree that it would be better to link to it from libunwind, if that is possible.)